src/jit/codegenxarch.cpp

```diff
@@ -1152,6 +1152,39 @@ void CodeGen::genReturn(GenTreePtr treeNode)
     // Also, there is not much to be gained by materializing it as an explicit node.
     if (compiler->compCurBB == compiler->genReturnBB)
     {
+#if defined(_TARGET_AMD64_)
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
This #ifdef is redundant since FEATURE_UNIX_AMD64_STRUCT_PASSING is defined only on amd64.